test/isochrone.cc

```diff
@@ -94,16 +103,27 @@ void test_isochrones() {
 // Test auto isochrone with one contour
 // 32bit builds fail in release mode we'll look at this separately
 #if _WIN64 || __amd64__
-  std::string expected1 = "\"type\":\"LineString\"";
-  const auto test_request1 =
-      R"({"locations":[{"lat":52.078937,"lon":5.115321}],"costing":"auto","contours":[{"time":15}],"polygons":false,"denoise":0.2,"generalize":150})";
-  try_isochrone(reader, loki_worker, thor_worker, test_request1, expected1);
-
-  // Try pedestrian isochrone with one contour, polygon=true
-  std::string expected2 = "\"type\":\"Polygon\"";
-  const auto test_request2 =
-      R"({"locations":[{"lat":52.078937,"lon":5.115321}],"costing":"bicycle","contours":[{"time":15}],"polygons":true,"denoise":0.2})";
-  try_isochrone(reader, loki_worker, thor_worker, test_request2, expected2);
+  {
+    std::cout << "test-case 1" << std::endl;
```

Comment:
Cleanup debug statements

Reply:
This was actually helpful when the test failed, it's not printed if test passes.

Reply:
> This was actually helpful when the test failed, it's not printed if test passes.

Shouldn't that be captured via a assert message (indicating what failed)?

Reply:
We're currently trying to switch to gtest (https://github.com/valhalla/valhalla/pull/2128 is 99% there) so there wasn't much point in using existing asserts when all of that goes away when #2128 is merged.

Reply:
Can you add a little more description to this `cout` statement, similar to what you have put in the next test following this one?

Reply:
I ended up removing this line.